lighthouse-core/gather/gatherers/accessibility.js

```diff
@@ -57,6 +57,8 @@ async function runA11yChecks() {
       'svg-img-alt': {enabled: false},
       'audio-caption': {enabled: false},
       'aria-treeitem-name': {enabled: true},
+      // https://github.com/dequelabs/axe-core/issues/2958
+      'nested-interactive': {enabled: false},
```

Comment:
'frame-focusable-content': {enabled: false}` too so we aren't wasting time on it

Reply:
> `'frame-focusable-content': {enabled: false}` too so we aren't wasting time on it

that was meant to be a question, I don't feel strongly about it :)

Not having axe tests running without any use of their output is the only real argument for disabling